x-pack/legacy/plugins/lens/public/metric_visualization_plugin/metric_config_panel.test.tsx

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { ReactWrapper } from 'enzyme';
+import { mountWithIntl as mount } from 'test_utils/enzyme_helpers';
+import { MetricConfigPanel } from './metric_config_panel';
+import { DatasourcePublicAPI, DatasourceDimensionPanelProps, Operation } from '../types';
+import { State } from './types';
+import { NativeRendererProps } from '../native_renderer';
+import { createMockDatasource } from '../editor_frame_plugin/mocks';
+
+describe('MetricConfigPanel', () => {
+  const dragDropContext = { dragging: undefined, setDragging: jest.fn() };
+
+  function mockDatasource(): DatasourcePublicAPI {
+    return {
+      ...createMockDatasource().publicAPIMock,
+      generateColumnId: () => 'TESTID',
+    };
+  }
+
+  function testState(): State {
+    return {
+      title: 'Test Metric',
+      accessor: 'foo',
+    };
+  }
+
+  function testSubj(component: ReactWrapper<unknown>, subj: string) {
+    return component
+      .find(`[data-test-subj="${subj}"]`)
+      .first()
+      .props();
+  }
+
+  test('allows editing the chart title', () => {
```

Comment:
I was testing out the behavior of this component and I'm a little surprised that the label doesn't default to the operation label. Is that a change you can make here?